src/libvirtApi/domain.js

```diff
@@ -138,6 +139,16 @@ export function domainAttachDisk({
     return domainAttachDevice({ connectionName, vmId, permanent, hotplug, xmlDesc });
 }
 
+export function domainAttachHostDevice({ connectionName, vmName, live, dev }) {
+    const source = getNodeDevSource(dev);
+    const update = live ? " --update" : "";
+
+    return cockpit.script(
+        `virt-xml -c qemu:///${connectionName} ${vmName} --add-device --hostdev ${source} ${update}`,
```

Comment:
See Martins comment in this other PR https://github.com/cockpit-project/cockpit-machines/pull/374#discussion_r709541242 
Same applies here.